packages/create-widget/views/uhtml/template/src/component/View.js

```diff
@@ -1,8 +1,13 @@
 import Welcome from './Welcome';
 import Counter from './Counter';
 import WidgetDescription from './WidgetDescription';
+import Error from './Error';
 
 export default function View(widget) {
+  if (widget.error && widget.error.status) {
+    return Error(widget);
+  }
+
   return widget.$dependencies.html`
```

Comment:
Neni to chyba? Vim ze to neni zmeneny kod, ale zarazil me ten zapis.

Reply:
Resp. co je v dependencies.html?

Reply:
nemela by se ta komponenta jmenovat Error, protoze to pak prebiji nativni Error.. opravim to..